agent/functional_tests/testdata/taskdefinitions/nvidia-gpu/task-definition.json

```diff
@@ -0,0 +1,14 @@
+{
+    "family": "ecsftest-nvidia-gpu",
+    "containerDefinitions": [{
+      "image": "nvidia/cuda:9.0-base",
+      "name": "exit",
+      "cpu": 100,
+      "memory": 100,
+      "resourceRequirements": [{
+        "type":"GPU",
+        "value": "2"
+      }],
+      "command": ["sh", "-c", "nvidia-smi -L | wc -l | grep \"2\" && exit 42 || exit 1"]
```

Comment:
`grep \"2\"` seems to be to broad, 20/22/12 will also pass the test, is there a way to check if it's exact 2?

Reply:
I think we can use `grep -w` for absolute match